mycity/mycity/utilities/gis_utils.py

```diff
@@ -7,8 +7,12 @@
 """
 
 from arcgis.features import FeatureLayer
-
 import mycity.utilities.google_maps_utils as g_maps_utils
+from arcgis.gis import *
+from arcgis.geocoding import geocode, reverse_geocode
+from math import sin, cos, sqrt, atan2, radians
+
+dev_gis = GIS()  # this is needed to use geocoding
```

Comment:
Do we know exactly what this is doing? Do we need a login to use this in production?